simplq/.eslintrc.json

```diff
@@ -7,7 +7,9 @@
     "node": true
   },
   "rules": {
-    "prettier/prettier": ["error"],
+    "prettier/prettier": ["error", {
+     "endOfLine":"auto"
+   }],
```

Comment:
What does this do?

Reply:
This is for VScode , It was giving some error in Editor so I just turned it off

Reply:
Don't change the config directly. In case you have an error you can't figure out, add an eslint comment to ignore the error for just that line or file. We can look at that later then.